aten/src/ATen/native/mps/operations/Distributions.mm

```diff
@@ -185,7 +185,7 @@
 
   @autoreleasepool {
     MPSShape* input_shape = getMPSShape(output);
-    string key = "normal_mps_out:" + getMPSShapeString(input_shape) + ":" + getMPSTypeString(output.scalar_type());
+    string key = "normal_mps_out:" + getMPSShapeString(input_shape) + ":" + getMPSTypeString(output.scalar_type()) + ":" + to_string(seed_);
```

Comment:
Thanks, we should pass in the seed. 
As a follow up we are going to update this API to use the seed to create Philox state which can be used to set the counter, low and high values.